### DISCUSSION TITLE
Release on github without assets

### DISCUSSION THREAD

**Discussion by bonddim** [helmfile/helmfile#976] [General]
@yxxhero ,
You should make release "published" only after all assets are uploaded to it, not before)
go-releaser can do this it automatically.

**Comment by yxxhero** [comment]
@bonddim PR is welcome.

**Reply by bonddim** [reply]
@yxxhero 
I made a [PR](https://github.com/helmfile/helmfile/pull/998), but it will not fix described issue :)

There are 2 options to create the release properly with GoReleaser:
1. Push tags
2. Create **pre-release** from GitHub UI - GoReleaser workflow will set it to latest on successful run.
